setup.py

```diff
@@ -0,0 +1,44 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="shopify-client",
+    version="0.1.0",
+    description="Python client for Shopify REST and GraphQL API",
+    long_description=open("README.md").read(),
+    long_description_content_type="text/markdown",
+    author="Anton Shutsik",
+    author_email="shutikanton@gmail.com",
+    url="https://github.com/groveco/shopify-client.git",
+    packages=["shopify_client"],
+    install_requires=[
+        "requests>=2.25.1",
+    ],
+    extras_require={
+        "dev": [
+            "pytest",
+            "pytest-cov",
+            "pytest-mock",
+            "flake8",
+            "black",
+            "sphinx",
+            "pre-commit"
+        ]
+    },
+    python_requires=">=3.7",
+    classifiers=[
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
```

Comment:
```suggestion
    },
    python_requires=">=3.9",
    classifiers=[
        "Programming Language :: Python :: 3",
```

Don't support [unsupported versions](https://devguide.python.org/versions/#supported-versions) of Python.